Controllers/ProductController.js

```diff
@@ -1,6 +1,7 @@
 const DbService = require("../Services/DbService");
 const ProductService = require("../Services/ProductsService");
 const JsonResponseService = require("../Services/JsonResponseService");
+const {sanitiseSku, sanitiseName, sanitiseStockLevel, sanitisePrice} = require("../Services/SanitiseValidateService");
```

Comment:
Don't need this line as it is facilitated by the one below and prepending the the function with 'DataCheckers.'